data/vm/vmOutputApi.go

```diff
@@ -52,7 +52,7 @@ type LogEntryApi struct {
 	Identifier []byte   `json:"identifier"`
 	Address    string   `json:"address"`
 	Topics     [][]byte `json:"topics"`
-	Data       []byte   `json:"data"`
+	Data       [][]byte `json:"data"`
```

Comment:
this is a breaking change that will affect all integrators that parse the logs. Suggestion to add a new field `AdditionalData` that contains a `[][]byte`

Reply:
+1